### PR TITLE
Add SMTP configuration for staging

### PR DESCRIPTION
**Pull request by enricostano**

### CODE REVIEW

File: roles/common/templates/default.j2 [hunk 1]
```diff
@@ -5,3 +5,8 @@ DATABASE_PASSWORD={{ database_password }}
 SECRET_TOKEN={{ secret_token }}
 MAIL_LINK_HOST=www.timeoverflow.org
 MAILER_SENDER={{ mailer_sender }}
+SMTP_ADDRESS=smtp.mailgun.org
+SMTP_PORT=587
+SMTP_DOMAIN=staging.timeoverflow.org
```
Comment (enricostano) [inline roles/common/templates/default.j2]: Use variable

File: inventory/host_vars/staging.timeoverflow.org/secrets.yml [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+30346437363130663262393736633736346639363838313366366362396261386631343339306234
+36343039643135363038653535633165346338383332626161393063333564626133653033633764
+32626530306538393130666438396166363262613065666535356338623639393963373464333034
+32623464383637376465386664653232336536626231656561666137303065633933316535616332
+31386365663837626232313364306230383562326138383639343930343033343034373861326136
+39633463333966656531653732383039353565333636356365326133663461326466663631386363
+32383966653932383634306166613962376436636165356266393231303937396334333032326535
+64376362336234373831356436393266653736353665386166393533626338363631303039386332
+38373637333335393261373835643964653936636131373263383136346163316666333661643864
+37306633313435373337
```
Comment (andresgutgon) [inline inventory/host_vars/staging.timeoverflow.org/secrets.yml]: No entiendo nada, no se ven las keys :trollface:

Comment (sauloperez) [reply inventory/host_vars/staging.timeoverflow.org/secrets.yml]: What a troll @andresgutgon

File: roles/common/templates/default.j2 (hunk 1 above)
Comment (danypr92) [reply roles/common/templates/default.j2]: By now, we only use a var for the SMTP_DOMAIN, the other vars are the same in the different environments.

**Pull request merged**